demo/tcf/ui/demo/index.js

```diff
@@ -0,0 +1,13 @@
+import '../../../../components/tcf/firstLayer/src/index.scss'
+import TcfUi from '../../../../components/tcf/ui/src'
+import React from 'react'
+
+const TcfUiDemo = () => {
+  return (
+    <>
+      <TcfUi />
```

Comment:
why don't you use the playground instead of adding the complete version?

Reply:
I've change it and I let it this way. Is there any problem to let it this way instead of with the playground? I can change it again if something will work better